src/layer/vector/Polyline.js

```diff
@@ -57,7 +57,11 @@ export const Polyline = Path.extend({
 
 		// @option noClip: Boolean = false
 		// Disable polyline clipping.
-		noClip: false
+		noClip: false,
+
+		// @option curved: Boolean = false
+		// Enable polyline Bézier curves. Curved polylines don't support clipping - each line is rendered fully.
```

Comment:
This needs to reflect that bezier curves are not supported when polylines are drawn on an `L.Canvas`.

Reply:
But that's supported. See changes in `src/layer/vector/Canvas.js`. SVG drawing commands are transformed into Canvas drawing commands in Canvas.js. I have also added vector-canvas-curved.html to test that.